notebooks/multi_modal/solutions/image_captioning.ipynb

```diff
@@ -0,0 +1,1136 @@
+{
```

Comment:
<p>Add a note above that the reason every line starts with 3 and ends with 4 is because, &lt;start&gt; gets mapped to 3 and &lt;end&gt; to 4</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/226/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Thanks! I added the line.</p>